src/with_ladj.jl

```diff
@@ -67,6 +70,21 @@ Implementations of with_logabsdet_jacobian can be tested (as a
 function with_logabsdet_jacobian end
 export with_logabsdet_jacobian
 
+
+
+"""
+    struct NoLogAbsDetJacobian{F,T}
+
+An instance `NoLogAbsDetJacobian{F,T}(f)` signifies that
+`with_logabsdet_jacobian(f, ::T)` is not defined.
+"""
+struct NoLogAbsDetJacobian{F,T} end
+export NoLogAbsDetJacobian
+
+with_logabsdet_jacobian(f::F, ::T) where {F,T} = NoLogAbsDetJacobian{F,T}()
```

Comment:
`f` could be removed:
```suggestion
with_logabsdet_jacobian(::F, ::T) where {F,T} = NoLogAbsDetJacobian{F,T}()
```